langchain/src/vectorstores/vectara.ts

```diff
@@ -27,6 +27,15 @@ export interface VectaraFilter {
   // between neural search and keyword-based search factors. Values between 0.01 and 0.2 tend to work well.
   // see https://docs.vectara.com/docs/api-reference/search-apis/lexical-matching for more details.
   lambda?: number;
+  // The number of sentences before/after the matching segment to add to the context.
+  contextConfig?: VectaraContextConfig;
+}
+
+export interface VectaraContextConfig {
+  // The number of sentences before the matching segment to add. Default is 2.
+  nBefore?: number;
```

Comment:
Maybe change this to nSentBefore and nSentAfter, so that it's clear this is related to the number of sentences not the number of characters (from the Vectara API)?